proof-producer/bin/proof-producer/src/arg_parser.hpp

```diff
@@ -47,6 +47,7 @@ namespace nil {
             boost::filesystem::path assignment_description_file_path;
             boost::filesystem::path challenge_file_path;
             boost::filesystem::path theta_power_file_path;
+            boost::filesystem::path evm_verifier_path = "evm_verifier";
```

Comment:
May be remove default value, and do print only if user set the output directory.
Otherwise we'll can't run proof-producer without print EVM verifier files, which affect performance

Reply:
@ETatuzova